api/src/main/java/com/cloud/network/NetworkPermission.java

```diff
@@ -0,0 +1,32 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+package com.cloud.network;
+
+import org.apache.cloudstack.api.InternalIdentity;
+
+public interface NetworkPermission extends InternalIdentity {
+
+    public enum Ops {
+        add,    // grant the access permission to a network
```

Comment:
@weizhouapache better to define the ops at some common point and use it across. also, check the code formatting for enum.

Reply:
@sureshanaparti 
It is not used in any place. I will remove it.